README.md

```diff
@@ -1,64 +1,90 @@
 
-- Setup the `~/.bash_aliases`
-```
-export ViconSDK_ROOT=location/of/the/ViconSDK/libraries
-export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${ViconSDK_ROOT}
-```
+## Dependencies
+
+- Get YARP from the [official GitHub repository](https://github.com/robotology/yarp.git)
+- Get ViconDataStreamSDK from the [official Vicon website](https://www.vicon.com/products/software/datastream-sdk)
 
+## Installation 
+
+### Linux
+- Setup the `~/.bash_aliases`
+  ```bash
+  export ViconSDK_ROOT=location/of/the/ViconSDK/libraries
+  export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${ViconSDK_ROOT}
+  ```
 - Configure and make
+  ```bash
+  $ git clone https://github.com/robotology/yarp-vicon-bridge.git
+  $ cd yarp-vicon-bridge
+  $ mkdir build && cd build
+  $ cd build 
+  $ ccmake ..
+  $ make install
+  ```
+- Check if the device has been installed correctly
+  ```bash
+  yarpdev --list | grep vicon
+  ```
 
-```
-$ git clone https://github.com/robotology/yarp-vicon-bridge.git
-$ cd yarp-vicon-bridge
-$ mkdir build && cd build
-$ cd build 
-$ ccmake ..
-$ make
-```
-System setup
-------------
-- Make sure the cameras are connected to the switch
-- Make sure that the pc where **Nexus** is running has the port 192.168.10.1 (bottom network card) connected to the switch (blue cable), and the black
-ethernet cable plugged in the top Network Card(check that the other side is connected to the *"r1 router"*).
+### Windows
+- Setup the environment variables
+  ```bash
+    setx ViconSDK_ROOT "location/of/the/ViconSDK/libraries"
+    setx PATH=%PATH%;ViconSDK_ROOT 
+  ```
+  :bulb: You can also set the environment variables using a GUI called [Rapid Environment Editor](https://www.rapidee.com/en/about)
+  
+- Clone this repository
+  ```bash
+  git clone https://github.com/robotology/yarp-vicon-bridge.git
+  ```
+- Configure and build the project using the [CMake GUI](https://cmake.org/runningcmake/) (:warning: set x64 for generator)
+- Build the project either using Visual Studio or the command line:
+  ```bash
+  cmake --build . --config Release --target install
+  ```
+  
+## How to run the `yarp-vicon-bridge`
 
-Run YarpViconBridge with R1 setup
----------------------------------
+### Using the Vicon setup
+
+- Run a `yarpserver` in a terminal
+- Open a second terminal and start a `transformServer` device:
+  ```bash
+  yarpdev --device transformServer --ROS::enable_ros_publisher 0 --ROS::enable_ros_subscriber 0
+  ```
+- Open a third terminal and start a `yarp-vicon-bridge` device:
+  ```bash
+  yarpdev --device vicon --hostname "192.168.10.2"
+  ```
+  :warning: the default `hostname` is `192.168.10.2` (see [Vicon Network Cards Settings](https://docs.vicon.com/display/Connect/Configuring+network+card+settings#Configuringnetworkcardsettings-IPAddresses)) but some network test might be needed to find out the port where the Vicon output is actually streamed.
+
+### Using the Vicon setup and R1
```

Comment:
It is not clear to me why R1 is mentioned here.
roscore can run on any machine with ROS installed.
yarpserver --ros can run on any machine with ROS installed.
R1 is not needed at all..

Reply:
Actually those ones were the original instructions available on the repository, and I didn't test myself on R1. So I might have misinterpreted them... 😅